integration/integration-api/src/main/java/se/inera/intyg/webcert/integration/api/subscription/SubscriptionRestService.java

```diff
@@ -16,11 +16,10 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package se.inera.intyg.webcert.integration.kundportalen.service;
+package se.inera.intyg.webcert.integration.api.subscription;
 
 import java.util.List;
 import java.util.Map;
-import se.inera.intyg.webcert.integration.kundportalen.enumerations.AuthenticationMethodEnum;
 
 public interface SubscriptionRestService {
```

Comment:
Should we call it SubscriptionIntegrationService instead of ...Rest...?

Reply:
done